rfcs/text/0000-rfc-process.md

```diff
@@ -0,0 +1,174 @@
+- A RFC is required if a design impacts the scope of multiple projects.
+- A RFC is required if a technical initiative (refactoring, major architectural change) impacts the community.
+- A RFC is required for all changes to DCP governance.  
+
+### Revising RFC(s)
+
+How are RFC(s) maintained as *living documents* in an iterative development process where the original design may substantially change based on implementation experiences?
+
+Where necessary, existing RFCs can be revised using the same process (a proposed change submitted as a pull request and subject to community review and approval). **Major changes should result in a new RFC.**
+
+### What are the roles in the RFC process?
+
+- **Authors** are DCP community members.
+
+- **Shepherds** are members of *DCP PM* assigned to assist the **Author(s)** and ensure that a RFC progresses to closure.
+
+- **Approvers** for governance RFC(s) are the Project Leads and Product Owners from *DCP PM*.
+
+- **Approvers** for software RFC(s) are the Technical Leads from *DCP Architecture*. 
+
+- **Reviewers** are DCP community members.
+
+- **Science Reviewer** is the *DCP Science PM* in coordination with *HCA Science Governance*.
+
+## Detailed Design
+
+### Proposing RFC(s)
+
+#### **Authors**:
+
+*Before proposing, always engage the DCP community to build consensus and assess whether your proposal sparks interest and addresses a need. You may even discover potential collaborators.*
+
+  - Fork the HumanCellAtlas dcp-community repository
+  - Copy `rfcs/rfc-template.md` to `rfcs/text/0000-my-feature.md`
+  *- where "my-feature" is descriptive. Don't assign a RFC number yet.*
+  
+  - Fill in the RFC with attention to detail
+
+  - Submit a pull request.
+  
+      For software RFC(s):
+      - Add the "rfc-proposed" and the appropriate software project name _(such as "Data Store")_ labels. When the RFC crosses the scope of software projects, then the "Architecture" label **MUST** be the project name. *If uncertain about the appropriate project name, then **_Ask a PM_** on the HCA **#dcp-project-mgmt** slack channel*
+
+      For governance RFCs:
+        
+      - Add the "rfc-proposed" and the "Governance" labels. 
+
+  - Share a link to the pull request for community review on the HumanCellAtlas **#dcp** slack channel
+
+#### DCP PM:
+- Assign a **Shepherd** by completing the *Shepherd* section in the RFC template and pushing the update. The **Shepherd** guides the RFC and its **Author(s)** through the process.
+- If there is a *Scientific "guardrails"* section in the RFC, assign the **Science Reviewer** as a reviewer of the pull request and add the "science-review-required" label.
+
+#### Shepherd:
+- Include the RFC submission in *This week in DCP* status update
+
+### Reviewing RFC(s)
+
+#### Approvers:
+- May assign specific reviewers in the RFC pull request
+
+#### Reviewers:
+- Add feedback to the pull request comment thread
+
+#### Science Reviewer:
```

Comment:
From a process perspective we need to make sure that RFCs that require scientific review don't get bottlenecked through one or a limited number of people. Doesn't necessarily change this document but worth thinking about.